doc/release/RELEASE-NOTES.md

```diff
@@ -28,6 +28,11 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-705](https://opensupplyhub.atlassian.net/browse/OSDEV-705) - Fixed the error “could not convert string to float” that occurred when a list contained columns for “lat” and “lng” and only some of the rows in these columns had data. As a result, rows are processed regardless of whether the values for “lat” and “lng” are present and valid, invalid, or empty.
 
 ### What's new
+* [OSDEV-981](https://opensupplyhub.atlassian.net/browse/OSDEV-981) Reporting. History of contributor uploads. Created a new report with details about the contributor:
+* including name, ID, contributor type;
+* first upload, including date of the first upload and time since the first upload in days;
+* most recent (or “last”) upload, including date of the last upload and time since the last upload in days;
+* total (or “lifetime”) uploads and a calculation for uploads per year (= lifetime uploads/time since the first upload (in years)). This data is ordered based on the “date of last upload” column so that contributors who have recently contributed data are at the top of the report.
```

Comment:
Wrong formatting
<img width="1415" alt="Screenshot 2024-07-05 at 12 52 45" src="https://github.com/opensupplyhub/open-supply-hub/assets/58369841/f90a147d-4171-44b4-a84b-946768f66936">

Reply:
Please refer to [the docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists) on how to create a nested list in the .md file. They use hyphens, but you can use asterisks as well.

Reply:
replaced with dash